src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1600,26 +1600,34 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.None
         End Function
 
-        Public Overrides Sub Accept(visitor As IOperationVisitor)
+        Public Overrides Sub Accept(visitor As OperationVisitor)
             Throw ExceptionUtilities.Unreachable
         End Sub
 
-        Public Overrides Function Accept(Of TArgument, TResult)(visitor As IOperationVisitor(Of TArgument, TResult), argument As TArgument) As TResult
+        Public Overrides Function Accept(Of TArgument, TResult)(visitor As OperationVisitor(Of TArgument, TResult), argument As TArgument) As TResult
             Throw ExceptionUtilities.Unreachable
         End Function
     End Class
 
     Partial Class BoundRaiseEventStatement
+        Implements IExpressionStatement
+
+        Public ReadOnly Property Expression As IExpression Implements IExpressionStatement.Expression
+            Get
+                Return Me.EventInvocation
+            End Get
+        End Property
+
         Protected Overrides Function StatementKind() As OperationKind
-            Return OperationKind.None
+            Return OperationKind.ExpressionStatement
```

Comment:
This is a little strange, in that there is a little more to the statement syntactically than an invocation expression, but semantically that's what it boils down to. I guess this is a good change. An explanatory comment would be helpful.